### ISSUE TITLE
QC: dfTransformed inconsistencies

### ISSUE THREAD

**Issue by jonthegeek** [Gilead-BioStats/gsm#1681]
## QC Details
The `Analyze_Identity()` help's "Data Specification" section says that `dfTransformed` for `Analyze_Identity` "is typically created using `Transform_Rate()`."

The parameter help for the same function says `dfTrasnformed` is created using `Transform_Count()`.

The `test` file for `Analyze_Identity()` uses `Transform_Rate()`.

Which one should be used? Is either one acceptable? While we're at it, can we double-check the preferred/required definitions of dfTransformed for each function that uses it? All the other ones say `Transform_Rate()`, but now I'm not sure whether that's a copy/paste bug.

If functions need one or the other, should we consider renaming the parameters (`dfTransformedRate` vs `dfTransformedCount`)?

- [ ] `Analyze_Fisher()`
- [ ] `Analyze_Identity()`
- [ ] `Analyze_NormalApprox()`
- [ ] `Analyze_NormalApprox_PredictBounds()`
- [ ] `Analyze_Poisson()`
- [ ] `Analyze_Poisson_PredictBounds()`
- [ ] `Analyze_QTL()`

## Additional Comments

**Comment by jwildfire** [comment]
Yeah, need to clean up and document approach here. I'll try to write up a summary soon-ish and then we can decide how to handle documentation. Might want to convert this to a discussion and the file issues for specific updates once we align on approach.

**Comment by jwildfire** [comment]
FWIW, a lot of this is (maybe confusing/unnecessary) future proofing, that can likely be ignored for the v2.0 release
